### PR TITLE
Document EasyCLA

### PR DESCRIPTION
**Pull request by NotMyFault**
I have taken some time to familiarize myself with the tooling provided by the Linux Foundation, covering EasyCLA too.

The past CLAs submitted have already been submitted through EasyCLA.
To streamline the process, I would recommend leaving existing CLAs in place, and guiding new CLA submissions to use EasyCLA, picking up Oleg's prior work.
Following this principle, this PR documents the process. The Linux Foundation already maintains a step-by-step guide with screenshots, which we can link to.

The submitter still creates a PR towards this repository with a dummy change to `pr.txt`, to trigger the EasyCLA bot. Once the CLA has been signed, the submitter can close the PR.
This workflow allows project maintainers to check whether a CLA has been signed or not, by browsing the PR tab and checking the bot's response.
This is as simple as checking the `collected/` directory for a signed CLA, what was used in the past to check the submitter status.

I left the PCC out of the documentation intentionally, given not everyone uses or has access to the LF PCC, therefore checking the bot’s response is easier, given it's public too.

Closes https://github.com/jenkinsci/infra-cla/pull/97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Jenkins project CLA
 
-The Jenkins project strives for a low barrier of entry. Therefore, we require a signed CLA for core committers and people belonging to specific teams only.
-If you are interested in joining the core, documentation, hosting, infrastructure, release or security team, a signed CLA is one prerequisite.
+The Jenkins project strives for a low barrier of entry. Therefore, we require a signed CLA for board members, officers, core committers and people belonging to specific teams only.  
+If you are interested in joining the core, documentation, hosting, infrastructure, release or security team, a signed CLA is one prerequisite.  
+If you are a board member or officer listed on the [governance page](https://www.jenkins.io/project/board/), you have signed a CLA already.
 
-Special interest groups and general participation in the project requires no signed CLA, although we welcome anyone who's willing to submit a CLA for their work.
+Special interest groups, regular or one-time committers and general participation in the project and requires no signed CLA, although we welcome anyone who's willing to submit a CLA for their work.
```
Comment (MarkEWaite) [inline README.md:7]: Attempt to simplify the phrasing and keep each of the descriptions as clearly identifying types of individuals.
```suggestion
Members of special interest groups, regular committers , general participants, and one-time committers in the project do not require a signed CLA, although we welcome anyone who's willing to submit a CLA for their work.
```

**Review by MarkEWaite — APPROVED** [review]
Optional phrasing refinement for your consideration

File: README.md (hunk 1 above)
Comment (NotMyFault) [reply README.md]: Thanks!

File: README.md [hunk 2]
```diff
@@ -1,39 +1,58 @@
...
-* Add a property file at `collected/icla/YOURNAME/committer.properties` and list your name, email address, and GitHub ID.
-* If your company has a signed CCLA in place, create a symlink at `collected/icla/YOURNAME/ccla` to the corresponding `../../ccla/COMPANY` directory.
-* File your fork as a pull request
+# How to submit a CLA
 
-There should be a plenty of examples of this in [our PR section](https://github.com/jenkinsci/infra-cla/pulls?q=is%3Apr+is%3Aclosed) .
+## Preamble
 
-To encrypt your CLA, you need gpg. First, obtain the public key of the Jenkins board:
+The Jenkins project uses the EasyCLA service provided by the Linux Foundation. We do no longer accept signed PDFs sent by email to the board or by a pull request to this repository.
```
Comment (MarkEWaite) [inline README.md:13]:
```suggestion
The Jenkins project uses the EasyCLA service provided by the Linux Foundation. We no longer accept signed PDFs sent by email to the board or by a pull request to this repository.
```

File: README.md [hunk 3]
```diff
@@ -1,39 +1,58 @@
...
 
-    curl https://raw.githubusercontent.com/jenkinsci/infra-cla/approved/publicKey.asc | gpg --import
+EasyCLA is a self service, which requires no manual intervention by the board or any other Jenkins project member. It is a one-time process, valid for all Jenkins [community spaces](https://www.jenkins.io/project/conduct/#community-spaces).
```
Comment (MarkEWaite) [inline README.md:15]:
```suggestion
EasyCLA is self service and requires no manual intervention by the board or any other Jenkins project member. It is a one-time process, valid for all Jenkins [community spaces](https://www.jenkins.io/project/conduct/#community-spaces).
```

**Review by MarkEWaite — COMMENTED** [review]
More optional comments.

**Review by oleg-nenashev — APPROVED** [review]
I would still publish public tables of people/orgs who signed the CLAs but looks good otherwise

**Pull request merged**